### ISSUE TITLE
Tooltip text/html adapter doesn't update

### ISSUE THREAD

**Issue by DanielScholte** [amcharts/amcharts5#1074]
I've used the [following example](https://codepen.io/team/amcharts/pen/ExXrXYo) from Amcharts.com as a reference to create a single tooltip for multiple series, which includes HTML.   
Using this example as a reference, I've ran into multiple issues that causes the adapter to stop triggering immediately or after a short while, causing the tooltip to show incorrect information.  
  
**HTML adapter**
The most important one is when setting the adapter of the forementioned example to `html` instead of `text`. This causes the adapter to trigger only once, and won't update after that.  
The following codepen shows this issue: https://codepen.io/Daniel-Scholte/pen/mdaPdOe  
  
**AutoTextColor**
Another issue that I've noticed is that the text adapter also stops to trigger after a short time (usually with a second) when the `autoTextColor` property of the tooltip is set to `false`.
The following codepen shows this issue: https://codepen.io/Daniel-Scholte/pen/bGOpbNP

**Environment (if applicable)**

- amCharts version 5.4.4
- Firefox version 117

**Comment by martynasma** [comment]
Here's a workaround for HTML:

```JavaScript
tooltip.on("x", function(x, tooltip) {
  tooltip.label.set("html", Math.random() + "");
});
```

And here's one for text-based with `autoTextColor: false`:

```JavaScript
tooltip.on("x", function(x, tooltip) {
  tooltip.label.set("text", Math.random() + "");
});
```
